setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+# D203  1 blank line required before class docstring
+# H201  no 'except:' at least use 'except Exception:'
+# H302  import only modules
+# H405  multi line docstring summary not separated with an empty line
+ignore = D203,H201,H302,H405
+# H106: Don’t put vim configuration in source files
+# H203: Use assertIs(Not)None to check for None
+enable-extensions = H106,H203
```

Comment:
All `Hxxx` are comming from OpenStack `hacking` https://docs.openstack.org/hacking/latest/ so it also need to be installed

Reply:
done